File: w06/d03/Classwork/makeTags.js
function makeTags (tag, str) {
	console.log('<' + tag + '>' + str + '</' + tag + '>')
};

makeTags('i', 'Yay')
// '<i>Yay</i>'
makeTags('i', 'Hello')
// '<i>Hello</i>'
makeTags('cite', 'Yay')
// '<cite>Yay</cite>'

File: w06/d03/Classwork/afternoon/make_tags.rb
=begin
The web is built with HTML strings like 
"Yay" which draws Yay as italic text. 
In this example, the "i" tag makes and 
which surround the word "Yay". Given tag 
and word strings, create the HTML string 
with tags around the word, e.g. "Yay".

javascript makeTags('i', 'Yay') // 
'<i>Yay</i>' makeTags('i', 'Hello') // 
'<i>Hello</i>' makeTags('cite', 'Yay') // 
'<cite>Yay</cite>'=end
=end



def maketags(tag, string)
	put "<#{tag}>#{string}</#{tag}>"
end

    